src/main/java/core/basesyntax/fileservice/ReadFromFile.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.fileservice;
+
+import core.basesyntax.fruitservice.Transaction;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFromFile {
```

Comment:
I think it's better to separate this class to several. Here you are reading from file, spliting lines and creating transactions. It looks good if every action is in own class